src/API/Site/Controllers/Jetpack/AccountController.php

```diff
@@ -114,6 +130,24 @@ protected function get_connect_callback(): callable {
 		};
 	}
 
+	/**
+	 * Get the query params for the connection request.
+	 *
+	 * @return array
+	 */
+	protected function get_connect_params(): array {
+		return [
+			'context' => $this->get_context_param( [ 'default' => 'view' ] ),
+			'next'    => [
+				'description'       => __( 'Indicate the next page name to map the redirect URI when back from Jetpack authorization.', 'google-listings-and-ads' ),
```

Comment:
Indicates* (3rd singular person)

Reply:
Suggestion-> Indicates the URL Callback after Jetpack authorization

Reply:
Adjusted in https://github.com/woocommerce/google-listings-and-ads/pull/1385/commits/cabe35a5033fec920e3d91bcd96c011569f9cc1a. The value is a page name but not a URL, so I kept "the next page name" in the description. Hope it's clearer now.